xen/drivers/passthrough/iommu.c

```diff
@@ -129,7 +129,7 @@ static void __init parse_iommu_param(char *s)
     } while ( ss );
 }
 
-int iommu_domain_init(struct domain *d)
+int iommu_domain_init(struct domain *d, bool_t use_iommu)
```

Comment:
Why this code is non-relevant to x86? The x86 needs the iommu_construct() like ARM. I just call this function a bit early